components/Navbar.tsx

```diff
@@ -10,8 +10,14 @@ import CategoryDropDownMenu from "./CategoryDropDownMenu";
 import AuthProfileMenu from "./AuthProfileMenu";
 
 const drawerWidth = 240;
-const navItems = ["Home", "Sign In", "Donation", "Mission", "Dashboard", "Contact"];
-const links = ["/", "/auth/sign-in", "/donation-info", "/mission-page", "/dashboard", "/contact-page"];
+
+// Removed "Dashboard" from navItems array
+const navItems = ["Home", "Sign In", "Donation", "Mission", "Contact"];
+// const navItems = ["Home", "Sign In", "Donation", "Mission", "Dashboard", "Contact"]; // Original
+
+// Removed "/dashboard" from links array
+const links = ["/", "/auth/sign-in", "/donation-info", "/mission-page", "/contact-page"];

```

Comment:
I was dealing with similar problem when sign in on nsc-event nextjs, i also think I solved. you can go to nsc-event  and branch is called Navbar updates #346 and compare. maybe that will be helpful

Reply:
@miqo1385 . Thanks. i will check that out.